lib/oneview-sdk/resource.rb

```diff
@@ -361,6 +361,12 @@ def recursive_like?(other, data = @data)
         return false unless data && data.respond_to?(:[])
         if val.is_a?(Hash)
           return false unless data.class == Hash && recursive_like?(val, data[key.to_s])
+        elsif val.is_a?(Array) && val.first.is_a?(Hash)
```

Comment:
@ricardogpsf is this going to be good for **all** resources and data sets? I imagined just adding a constant to certain Resource classes with the key names that we should ignore the order of array elements. I wonder if this may have farther reaching negative effects where the array ordering is important.